chain-network/proto/watch.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+
+import "types.proto";
+
+package iohk.chain.watch;
+
+message BlockSubscriptionRequest {}
+
+message TipSubscriptionRequest {}
+
+message MempoolSubscriptionRequest {}
+
+message SyncMultiverseRequest {
+  // Block IDs to start synchronization from.
+  //
+  // The client should include the ID of the latest block at or below the
+  // stability depth, as well as tips of any branches known to it from
+  // previously synchronized node state.
+  //
+  // The server should prune the block IDs in the list, discarding any block IDs
+  // not found in the blockchain storage (which might be on discarded branches),
+  // as well as any blocks that are ancestors of other blocks identified in
+  // the list. The server responds with a stream sending any
+  // valid blocks on the (possibly forked) chain that are descendants of
+  // the remaining blocks in the list, unless the list is empty, in which case
+  // the server sends all blocks starting from genesis.
```

Comment:
if `from` is empty then it will behave like `GetAllBlocks`, so I'm not sure if it's worth having them as two separate methods.

> Why not just from the latest confirmed(at stability depth) block?

Whose latest confirmed block? I don't see how you would use it if you mean the what the considers the confirmed block. You need the ancestors of it anyway.

Reply:
>This is to avoid the situation in which the node just switched the tip in a shallow fork and the server will stream again the whole chain.

Why does the tip matter here? The behavior shouldn't change regardless.

Reply:
> if `from` is empty then it will behave like `GetAllBlocks`,  so I'm not sure if it's worth having them as two separate methods.

This may stems from a misunderstanding about the meaning of "ID of the latest block at or below the stability depth", but the purpose of having two separate methods is indeed to avoid it behaving like `GetAllBlocks` if `from` ended up empty for futile causes (e.g. all branches provided have been gc'ed) 

> Why does the tip matter here? The behavior shouldn't change regardless.

Bad choice of word from my side. I meant to described the case where all block ids provided pointed to stale blocks which may have been eliminated already. (Again, this may be not be the case if I misunderstood the first part)